assemblies/assembly-building-custom-dashboard-widgets.adoc

```diff
@@ -1,21 +1,38 @@
 [id='assembly-building-custom-dashboard-widgets']
-= Building custom dashboard widgets
+= Creating custom pages in {CENTRAL}
 ifdef::context[:parent-context: {context}]
 // Context attribute is assembly specific and enables module reuse between assemblies.
 :context: building-custom-dashboard-widgets
 
 // Purpose statement for the assembly
-As a business analyst or business rules developer, use the *Page Editor* tool to design and define your page and specify the information to be displayed on it. You can decide which components are a part of your page. You can also customize the component properties of a page according to your requirements. Optionally, before creating a page, you can define a data set to feed the reporting components of the page.
+As a business analyst or business rules developer, you can use the *Page Editor* in {CENTRAL} to create static pages and dynamic dashboards to display specific information about your project. Dashboards are pages that contain at least one dynamic reporting component. You can define a data set to feed the reporting components of a dashboard. You can export dashboards to the standalone Dashbuilder Runtime viewer on {EAP} or the Dashbuilder Standalone viewer on {OPENSHIFT}.
```

Comment:
Is there a new convention to call "Dashbuilder Runtime" "Dashbuilder Runtime viewer"? I have not seen this before and in other places of the documentation, we don't have such wording.

Reply:
Clarified as Dashbuilder Runtime dashboard viewer. We need to tell users what DR and DS are. They are dashboard viewers.